engine/apps/labels/tasks.py

```diff
@@ -162,3 +162,56 @@ def update_instances_labels_cache(organization_id: int, instance_ids: typing.Lis
             continue
         if label_option:
             update_label_option_cache.apply_async((label_option,))
+
+
+@shared_dedicated_queue_retry_task(autoretry_for=(Exception,), retry_backoff=True, max_retries=MAX_RETRIES)
+def add_service_label_for_alerting_integrations():
+    """Starts tasks that add `service_name` dynamic label to custom labels for alerting integrations"""
```

Comment:
Please, add comment that it's intended for 1-time run. Also, let's stop use term **custom labels**